deploy/camel-catalog-2.23.0.yaml

```diff
@@ -1,20 +1,3 @@
-# ---------------------------------------------------------------------------
```

Comment:
If it's generated, we could treat consider to add the license header automatically.